workspaces/global-floating-action-button/packages/app/package.json

```
@@ -0,0 +1,83 @@
+{
+  "name": "app",
+  "version": "0.0.1",
+  "private": true,
+  "bundled": true,
+  "backstage": {
+    "role": "frontend"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redhat-developer/rhdh-plugins",
+    "directory": "workspaces/global-floating-action-button/packages/app"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "test": "backstage-cli package test",
+    "lint": "backstage-cli package lint"
+  },
+  "dependencies": {
+    "@backstage/app-defaults": "^1.5.12",
+    "@backstage/catalog-model": "^1.7.0",
+    "@backstage/cli": "^0.28.0",
+    "@backstage/core-app-api": "^1.15.1",
+    "@backstage/core-components": "^0.15.1",
+    "@backstage/core-plugin-api": "^1.10.0",
+    "@backstage/integration-react": "^1.2.0",
+    "@backstage/plugin-api-docs": "^0.11.11",
+    "@backstage/plugin-catalog": "^1.24.0",
+    "@backstage/plugin-catalog-common": "^1.1.0",
+    "@backstage/plugin-catalog-graph": "^0.4.11",
+    "@backstage/plugin-catalog-import": "^0.12.5",
+    "@backstage/plugin-catalog-react": "^1.14.0",
+    "@backstage/plugin-org": "^0.6.31",
+    "@backstage/plugin-permission-react": "^0.4.27",
+    "@backstage/plugin-scaffolder": "^1.26.1",
+    "@backstage/plugin-search": "^1.4.18",
+    "@backstage/plugin-search-react": "^1.8.1",
+    "@backstage/plugin-techdocs": "^1.11.0",
+    "@backstage/plugin-techdocs-module-addons-contrib": "^1.1.16",
+    "@backstage/plugin-techdocs-react": "^1.2.9",
+    "@backstage/plugin-user-settings": "^0.8.14",
+    "@backstage/theme": "^0.6.0",
+    "@mui/icons-material": "5.15.17",
+    "@mui/lab": "5.0.0-alpha.173",
+    "@mui/material": "5.15.17",
+    "@mui/styles": "5.16.7",
+    "@red-hat-developer-hub/backstage-plugin-global-floating-action-button": "^0.1.0",
```

This should be workspace dependency so that it doesn't use the released package from npm:

```suggestion
    "@red-hat-developer-hub/backstage-plugin-global-floating-action-button": "workspace:^",
```